controller.rst

```diff
@@ -146,11 +118,12 @@ and many others that you'll learn about next.
 
 .. tip::
 
-    You can extend either ``Controller`` or ``AbstractController``. The difference
-    is that when you extend ``AbstractController``, you can't access services directly
-    via ``$this->get()`` or ``$this->container->get()``. This forces you to write
-    more robust code to access services. But if you *do* need direct access to the
-    container, using ``Controller`` is fine.
+    What's the difference between ``Controller`` or ``AbstractController``. Not much:
+    both are identical, except that ``AbstractController`` is more restrictive: it
```

Comment:
I don't like this AbstractController vs Controller thing :(  For this introductory article is confusing and it forces the user to make a decision they cannot make at this point (because they know nothing about services and container and get(...)).

But at the same time this is probably the best place to talk about this ... so I'm really confused!